test/e2e/play_kube_test.go

```diff
@@ -396,4 +404,14 @@ var _ = Describe("Podman generate kube", func() {
 		Expect(logs.ExitCode()).To(Equal(0))
 		Expect(logs.OutputToString()).To(ContainSubstring("Operation not permitted"))
 	})
+
+	It("podman play kube with pull policy of never should be 125", func() {
```

Comment:
this is a great start. would you mind adding a couple more tests of the other options, so if they change behavior we catch it here too?